api/src/main/java/org/pmiops/workbench/api/OfflineRuntimeController.java

```diff
@@ -338,7 +357,29 @@ private boolean notifyForUnusedDisk(LeonardoListPersistentDiskResponse disk, int
     }
 
     mailService.alertUsersUnusedDiskWarningThreshold(
-        dbUsers, workspace.get(), disk, daysUnused, initialCreditsRemaining);
+        dbUsers, workspace.get(), disk, diskStatus, daysUnused, initialCreditsRemaining);
     return true;
   }
+
+  private String getDiskStatus(
+      LeonardoListPersistentDiskResponse diskResponse, String googleProject) throws ApiException {
+    if (leonardoMapper.toApiListDisksResponse(diskResponse).getIsGceRuntime()) {
+      // There could be just one runtime per google project, so set disk status as Attached if
```

Comment:
I don't think this logic is correct - see https://github.com/all-of-us/workbench/pull/7771#discussion_r1232499231

@NehaBroad